src/components/forward/forward.js

```diff
@@ -37,7 +37,11 @@ const mapStateToProps = state => ({
  * @returns {Object} - The object translations
  */
 const translates = (props: any) => ({
-  forwardText: !props.step ? <Text id={'controls.forward'}></Text> : <Text id={'controls.secondsForward'} fields={{seconds: props.step}}></Text>
+  forwardText: !props.step ? (
+    <Text id={'controls.secondsForward'} fields={{seconds: FORWARD_DEFAULT_STEP}}></Text>
+  ) : (
+    <Text id={'controls.secondsForward'} fields={{seconds: props.step}}></Text>
```

Comment:
if the id value is the same and only the fields value is changing according to the prop, so you can do something like:
`forwardText: <Text id={'controls.secondsForward'} fields={{seconds: props.step || FORWARD_DEFAULT_STEP}}></Text>`